src/main/java/net/minecraftforge/installertools/SrgMcpRenamer.java

```diff
@@ -49,21 +51,22 @@
 import net.minecraftforge.installertools.util.Utils;
 
 public class SrgMcpRenamer extends Task {
-
     @Override
     public void process(String[] args) throws IOException {
-
         OptionParser parser = new OptionParser();
         OptionSpec<File> mcpO = parser.accepts("mcp").withRequiredArg().ofType(File.class).required();
         OptionSpec<File> inputO = parser.accepts("input").withRequiredArg().ofType(File.class).required();
         OptionSpec<File> outputO = parser.accepts("output").withRequiredArg().ofType(File.class).required();
+        OptionSpec<Boolean> stripSignatures0 = parser.accepts("strip-signatures").withRequiredArg().ofType(Boolean.class);
 
         try {
             OptionSet options = parser.parse(args);
 
             File mcp = options.valueOf(mcpO).getAbsoluteFile();
             File input = options.valueOf(inputO).getAbsoluteFile();
             File output = options.valueOf(outputO).getAbsoluteFile();
+            Boolean stripSignaturesOption = options.valueOf(stripSignatures0);
```

Comment:
has should be enough, no need for an explicit argument. If it's not in it's false.